src/routing.ts

```diff
@@ -0,0 +1,47 @@
+export type RouteCategories = 'Statystyki' | 'Menu' | 'Magazyn' | 'Dostęp' | 'Ustawienia';
+
+export interface IRouteData {
+  path: string;
+  name: string;
+}
+
+export type Route<T> = {
+  [P: string]: T;
+};
+
+export const ROUTE: Route<IRouteData> = { HOME: { path: '/', name: 'Strona główna' } };
+
+export const ROUTE_STATS: Route<IRouteData> = {
+  SALES: { path: '/statistics/sales', name: 'Sprzedaż' },
+  CUSTOMERS: { path: '/statistics/customers', name: 'Klienci' },
```

Comment:
Sometimes you don't append `/` sign at the end of path and sometimes you do.
Let's stick to one pattern.

Reply:
I fixed this by adding a slash to each path.